.github/workflows/pylint.yml

```diff
@@ -0,0 +1,42 @@
+name: Pylint
+
+on: 
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+
+    - name: Run Lint
+      working-directory: server
+      run: |
+        REPORT=$(pylint --exit-zero `ls -R|grep .py$|xargs`)
+        echo "$REPORT"
+        echo ::set-env name=EXPOSED_REPORT::"${REPORT//$'\n'/'%0A%0A'}"
+
+    - name: Comment
```

Comment:
What's that part doing, is it adding the report to Github? and if so is it under the pull request?

Reply:
I tried to answer this question above (the same should apply here)